README.md

```diff
@@ -50,74 +47,70 @@ This task overrides the path to the global `datadog-ci.config.json` file.
 ```yml
 - datadog-mobile-app-upload@1:
    inputs:
-   - api_key: $DATADOG_API_KEY
-   - app_key: $DATADOG_APP_KEY
+   - api_key: <DATADOG_API_KEY>
+   - app_key: <DATADOG_APP_KEY>
    - config_path: './synthetics-config.json'
 ```
 
-For an example configuration file, see the [`global.config.json` file][2001].
+For an example configuration file, see the [`global.config.json` file][7].
 
 ### Example including all possible configurations
 
-For reference here's how a full configuration could look:
+For reference, this is an example of a complete configuration:
 
 ```yml
 - datadog-mobile-app-upload@1:
    inputs:
-   - api_key: $DATADOG_API_KEY
-   - app_key: $DATADOG_APP_KEY
+   - api_key: <DATADOG_API_KEY>
+   - app_key: <DATADOG_APP_KEY>
    - config_path: './global.config.json'
    - latest: true
    - mobile_application_version_id: '123-123-123'
    - mobile_application_version_file_path: 'path/to/application.apk'
-   - site: 'datadoghq.com'
+   - site: {{< region-param key="dd_site" code="true" >}}
```

Comment:
Could we avoid using `{{ }}` markup in publicly available markdown documents? It'll be displayed as is. Instead I believe we should use `<!-- -->` markup, like [on the datadog-ci synthetics readme](https://github.com/DataDog/datadog-ci/blob/bff12e212215133d0eb59ae1814d052a2ee886c2/src/commands/synthetics/README.md?plain=1#L68).
However, because this line is within a code block, I don't think the markup would work, so I'd rather not have anything here, and keep the simple `datadoghq.com`.
```suggestion
   - site: 'datadoghq.com'
```